### DISCUSSION TITLE
GSoC 2025 program still not officially announced

### DISCUSSION THREAD

**Discussion by orbeckst** [MDAnalysis/mdanalysis#4843] [GSoC Discussions]
Google still has not announced GSOC 2025. We got a note through the GSOC Mentors list that they hope to do so in early 2025. 

Given that organizations (like MDAnalysis) first have to apply to participate in GSOC , it is quite possible that the GSOC time line will shift compared to what it looked like in previous years. However, we do not know more than that.

We will let you know once we know more.

**Comment by umairgcu** [comment]
I am eager to contribute to the open-source community and enhance my skills through participation in Google Summer of Code. I am particularly interested in projects related to the analysis of molecular simulation data using Python.
MDAnalysis contributing has been to GSoC since 2020 and I am excited to explore potential opportunities to contribute to their impactful work in the upcoming GSoC 2025 program.

**Comment by orbeckst** [comment]
GSOC 2025 has been announced https://opensource.googleblog.com/2025/01/google-summer-of-code-2025-is-here.html and MDAnalysis will apply.

Until MDAnalysis has officially been accepted as GSOC 2025 Org, there's not much else we can tell you. You can have a look at last year's blog post https://www.mdanalysis.org/2024/02/27/gsoc2024/ to get an idea how this year might look, although keep in mind that we may make some changes to the procedure.

**Comment by orbeckst** [comment]
We applied for GSOC 2025, see #4916 .
